mis_builder/datas/ir_cron.xml

```diff
@@ -7,9 +7,8 @@
             <field name="interval_type">hours</field>
             <field name="numbercall">-1</field>
             <field eval="False" name="doall"/>
-            <field eval="'mis.report.instance'" name="model"/>
-            <field eval="'_vacuum_report'" name="function"/>
-            <field eval="'(24,)'" name="args"/>
+            <field ref="model_mis_report_instance" name="model_id"/>
+            <field name="code">model._vacuum_report(24, 0)</field>
```

Comment:
`hours=24` (default, BTW). The second number is? I don't see it here https://github.com/OCA/mis-builder/blob/fb3654e75551a8d74b8b73b1aec8ac6b2c795816/mis_builder/models/mis_report_instance.py#L466

Reply:
@simahawk fixed